### PR TITLE
Local juel

### PR DESCRIPTION
**Pull request by talentek**
Fixed code changes done by Nafis. Also added back the original Greetings class. Please review and approve these changes.

### CODE REVIEW

File: .classpath [hunk 1]
```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<classpath>
```
Comment (sayedmhasan15) [inline .classpath:2]: why is that ?

Comment (sayedmhasan15) [reply .classpath:2]: ??

Comment (talentek) [reply .classpath:2]: This change is to get the Maven update

**Comment by sayedmhasan15** [comment]
approved

**Pull request merged**